### PR TITLE
GOB-1259 Add begin_geldigheid and eind_geldigheid to relations. Updat…

### PR DESCRIPTION
**Pull request by JKoetsier**
…e model with extra test entities. Handle None values correctly in query_file.

### CODE REVIEW

File: gobcore/database/reader/file.py [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
 
     The pandas library is used to iterate through the items
 
     :return: a list of dicts
     """
+    def convert_row(row):
+        def convert(v):
+            if isinstance(v, float):
+                return None if pandas.isnull(v) else v
+            return v
```
Comment (Vegter) [inline gobcore/database/reader/file.py]: Remove  float instance test

File: gobcore/database/reader/file.py [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+
+        return {k: convert(v) for k, v in row.items()}
 
     for _, row in connection.iterrows():
+        row = convert_row(row)
         yield row
```
Comment (Vegter) [inline gobcore/database/reader/file.py]: yield convert_row(row) ?

File: gobcore/database/reader/file.py (hunk 1 above)
Comment (JKoetsier) [reply gobcore/database/reader/file.py]: Done

File: gobcore/database/reader/file.py (hunk 2 above)
Comment (JKoetsier) [reply gobcore/database/reader/file.py]: Looks better indeed

**Pull request merged**